src/parser/antlr_line_buffered_input.cpp

```diff
@@ -326,43 +295,42 @@ antlr3LineBufferedStreamNew(std::istream& in, ANTLR3_UINT32 encoding, pANTLR3_UI
 
     // Now we can set up the file name
     //
-    input->istream->streamName	= input->strFactory->newStr8(input->strFactory, name);
-    input->fileName		= input->istream->streamName;
+    input->istream->streamName =
+        input->strFactory->newStr8(input->strFactory, name);
+    input->fileName = input->istream->streamName;
 
     return input;
 }
 
-static pANTLR3_INPUT_STREAM
-antlr3CreateLineBufferedStream(std::istream& in)
-{
-	// Pointer to the input stream we are going to create
-	//
-	pANTLR3_INPUT_STREAM    input;
-
-	if	(!in)
-	{
-		return NULL;
-	}
-
-	// Allocate memory for the input stream structure
-	//
-	input   = (pANTLR3_INPUT_STREAM)
-		ANTLR3_CALLOC(1, sizeof(ANTLR3_LINE_BUFFERED_INPUT_STREAM));
-
-	if	(input == NULL)
-	{
-		return	NULL;
-	}
-
-	// Structure was allocated correctly, now we can install the pointer
-	//
-        input->data             = malloc(1024);
-        input->isAllocated	= ANTLR3_FALSE;
-
-        ((pANTLR3_LINE_BUFFERED_INPUT_STREAM)input)->in = &in;
-	// Call the common 8 bit input stream handler
-	// initialization.
-	//
+static pANTLR3_INPUT_STREAM antlr3CreateLineBufferedStream(
+    std::istream& in, LineBuffer* line_buffer) {
+  // Pointer to the input stream we are going to create
+  //
+  pANTLR3_INPUT_STREAM input;
+
+  if (!in) {
+    return NULL;
+  }
+
+  // Allocate memory for the input stream structure
+  //
+  input = (pANTLR3_INPUT_STREAM)ANTLR3_CALLOC(
+      1, sizeof(ANTLR3_LINE_BUFFERED_INPUT_STREAM));
+
+  if (input == NULL) {
+    return NULL;
+  }
+
+  // Structure was allocated correctly, now we can install the pointer
+  //
+  input->data = malloc(1024);
```

Comment:
Oops, this is a leftover from the original code. I had it fixed locally after writing the commit message and then forgot to include the change in the pull request, sorry.